src/assessment.js

```diff
@@ -49,5 +49,6 @@ var q2 = [/* your answer here */];
 // executing it.
 //
 var q5 = function fireOnAlderaan () {
-  // your code here
+  deathStar.fire.bind(alderaan);
```

Comment:
Very close - when you use `bind`, you need to invoke the result (`deathStar.fire.bind(alderaan)`) in order to actually execute the function that you've bound.